DQMOffline/Trigger/python/ExoticaMonitoring_Client_cff.py

```diff
@@ -15,6 +15,37 @@
   
 )
 
+NoBPTXEfficiency = cms.EDAnalyzer("DQMGenericClient",
```

Comment:
Ok, I changed it, and I assumed I should also change it for the metEfficiency module in ExoticaMonitoring_Client_cff.py as well, but that apparently gives conflicts... please let me know what I should do.